### PR TITLE
Add unseatCandidates script

### PR DESCRIPTION
**Pull request by PepijnSenders**
Reduced the amount of candidates from 181 to 43 🎉 

screen shot 2017-07-12 at 18 28 42

### CODE REVIEW

File: src/tasks/unseatCandidates.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+const getOutsideCollaborators = async (client, config) => {
+    const { data } = await client.get(`/repos/${config.owner}/${config.repo}/collaborators?affiliation=outside`);
+
+    return data;
+};
+
+const removeCollaborator = async (client, config) =>
+    client.delete(`/orgs/${config.owner}/outside_collaborators/${config.candidateUsername}`);
+
+const deleteRepositoryIfInactive = async (client, config) => {
```
Comment (kieranajp) [inline src/tasks/unseatCandidates.js]: Do we want to do this? I think we'd rather keep the repos for reference.

Comment (PepijnSenders) [inline src/tasks/unseatCandidates.js]: Naming messed up here, that was the idea before. But now we just unseat. Removing is optional.

Comment (kieranajp) [inline src/tasks/unseatCandidates.js]: I don't think deleting should even be an option tbh, otherwise someone will do it by accident...

Comment (PepijnSenders) [inline src/tasks/unseatCandidates.js]: Haha how? Check the screen 😛

Comment (PepijnSenders) [inline src/tasks/unseatCandidates.js]: Or should I change it to `if (false)` 😸

Comment (kieranajp) [inline src/tasks/unseatCandidates.js]: Well, they'll say 'yes' and then realise they didn't mean to, at which point there's no way back.

Copy/pasting my comments on previous tests has been useful when grading other ones, to speed me up 😉 so I really wouldn't want someone to delete them.

Just get rid of this whole bit :P https://github.com/hellofresh/normal-github-client/pull/2/files#diff-e869c77e0d4522ea1980fbb18e639e34R83

**Pull request merged**